src/components/MenuBar.js

```diff
@@ -326,9 +326,9 @@ class MenuBar extends React.Component {
         </style>
         {this.state.sticky && stickyCTA && cta}
         {!collective.isArchived && ['COLLECTIVE', 'EVENT'].indexOf(collective.type) !== -1 && (
-          <Link className="submitExpense darkBackground" href={`${collective.path}/expenses/new`}>
```

Comment:
No, this was working. Why change?

Reply:
The selector I got when I inspected was different, or should I still leave it using <Link>?

Reply:
Yes, it was fine with `<Link>`.

You should not confuse the React source code and the HTML generated code.

In this case, the React `<Link>` creates an HTML `<a>`.